browser/src/slideshow/LayerDrawing.ts

```diff
@@ -207,6 +207,13 @@ class LayerDrawing {
 	}
 
 	private drawVideos(slideHash: string) {
+		if (
+			!this.layerRenderer.getRenderContext().is2dGl() &&
+			!VideoRendererGl.videoProgramInitialized
+		) {
+			VideoRendererGl.createProgram(this.layerRenderer.getRenderContext());
+		}
+
```

Comment:
I'm confused as to why this change has been made - is it to avoid program creation for videos that never draw? The previous code potentially caught errors that this replacement doesn't. If `createProgram` fails, we now have an unhandled exception with no fallback.

Reply:
ok, sounds good - thanks for the explanation :)